script.js

```diff
@@ -1,12 +1,543 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+// level 500
+function getAllTvShowsApi() {
+  const rootDiv = document.getElementById("root");
+  const mainContainer = document.createElement("div");
+  mainContainer.setAttribute("id", "mainContainer");
+  rootDiv.appendChild(mainContainer);
+  const allTvShowData = getAllShows().sort(compare);
+  tvShowHeaderFooter();
+  allTvShowData.forEach((tvShow) => {
+    mainContainer.appendChild(allTvShowsListings(tvShow));
+  });
 }
 
-function makePageForEpisodes(episodeList) {
+function homeReturnBtn() {
+  //button which returns user home
+  let btn = document.createElement("button");
```

Comment:
I believe this could be a `const` and this applies to the majority of variables in the code. A good rule is to always use `const` and only change it to `let` if you really need to change that variable.